uber/models.py

```diff
@@ -1523,16 +1544,44 @@ def weighted_hours(self):
     def total_hours(self):
         return self.weighted_hours * self.slots
 
-    @cached_property
-    def all_staffers(self):
-        return self.session.query(Attendee).order_by(Attendee.last_first).all()
```

Comment:
Yeah, I generally expect IDEs to be pretty good at this kind of rename.  Renaming something like a database column would be harder/impossible to do automagically, since that also shows up in page handler parameters and in HTML form fields and test data dumps, etc.